### ISSUE TITLE
Segmentation fault on torch.addmm

### ISSUE THREAD

**Issue by andrew12678** [pytorch/pytorch#37143]
## 🐛 Bug

I was doing a simple transfer learning workflow (eg. I set up the dataset, dataloader and model) and then fed the batch of images to my model and I get a Seg Fault. I did a `export PYTHONFAULTHANDLER=1`, to track it down to the following line of code: `ret = torch.addmm(bias, input, weight.t()`in `torch/nn/functional.py`.

The full error message is as follows.

```
andrew@Andrews-iMac-Pro test % python3 test.py
Fatal Python error: Segmentation fault

Current thread 0x000000010833bdc0 (most recent call first):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torch/nn/functional.py", line 1370 in linear
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torch/nn/modules/linear.py", line 87 in forward
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torch/nn/modules/module.py", line 532 in __call__
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torchvision/models/resnet.py", line 211 in _forward_impl
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torchvision/models/resnet.py", line 216 in forward
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/torch/nn/modules/module.py", line 532 in __call__
  File "test.py", line 12 in <module>
zsh: segmentation fault  python3 test.py
```

## To Reproduce

I swtiched from my own data to the following example (slightly borrowed from the PyTorch docs) to reproduce the problem and it does still persist (with the exact same error message as well).

1. Run the following the code
```python
import torch
import torchvision
from torchvision import datasets, transforms

transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.5,), (0.5,))])
trainset = datasets.MNIST('mnist_train', train=True, download=True, transform=transform)

trainloader = torch.utils.data.DataLoader(trainset, batch_size=4, shuffle=True)
model = torchvision.models.resnet50(False)
model.conv1 = torch.nn.Conv2d(1, 64, kernel_size=7, stride=2, padding=3, bias=False)
images, labels = next(iter(trainloader))
output = model(images)
```

## Expected behavior

I expect to see some valid ResNet50 output from the MNIST data.

## Environment

 - PyTorch Version (e.g., 1.0): 1.5.0 (also errorenous on 1.4.0 as well) 
 - OS (e.g., Linux): Mac OSX 10.15.4 (Hackintosh)
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Python version: 3.8.2
 - CUDA/cuDNN version: None
 - GPU models and configuration: Radeon RX570 (I don't believe this matters since I haven't used `.cuda()` anywhere)
 - Any other relevant information: My CPU is an AMD Ryzen 5 2600 (which I don't think matters because I'm using `MKL_DEBUG_CPU_TYPE=5`)

## Additional context

Versions of relevant libraries:
[pip3] numpy==1.18.2
[pip3] torch==1.5.0
[pip3] torchvision==0.6.0
[conda] blas                      1.0                         mkl  
[conda] mkl                       2019.4                      233  
[conda] mkl-service               2.3.0            py37hfbe908c_0  
[conda] mkl_fft                   1.0.15           py37h5e564d8_0  
[conda] mkl_random                1.1.0            py37ha771720_0  
[conda] numpy                     1.18.1           py37h7241aed_0  
[conda] numpy-base                1.18.1           py37h6575580_1  
[conda] numpydoc                  0.9.2                      py_0

cc @malfet @albanD @jianyuh @nikitaved @pearu @mruberry @heitorschueroff @walterddr @IvanYashchuk @xwang233 @Lezcano @jeffdaily @sunway513 @vincentqb @vishwakftw

**Comment by zhangguanheng66** [comment]
@fmassa could you take a look at here?

**Comment by zhangguanheng66** [comment]
Cannot reproduce on my side

**Comment by andrew12678** [comment]
@zhangguanheng66 @xwang233 is there any further diagnostics I can run from my end to see why the segmentation fault is occuring?

It seems like other people on other operating systems (Ubuntu) have been effected in the past but haven't come up with a solution yet.

https://stackoverflow.com/questions/58748553/segmentation-fault-in-pytorch

https://stackoverflow.com/questions/54210933/when-i-code-torch-addmm-in-pytorch-and-then-segmentation-fault-why

**Comment by jeffdaily** [comment]
Removed rocm label since GPU is not used in the reproducer code.

**Comment by mruberry** [comment]
Closing due to age and lack of reproduction. If this is still occurring with recent versions of PyTorch please file a new issue and we'll try again to reproduce it.
